code/game/objects/items.dm

```diff
@@ -60,6 +61,8 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 	var/block_chance = 0
 	var/hit_reaction_chance = 0 //If you want to have something unrelated to blocking/armour piercing etc. Maybe not needed, but trying to think ahead/allow more freedom
 
+	var/toolspeed = 1 // If this item is a tool, the speed multiplier

```

Comment:
I believe /tg/ initially added it as a divider, but because multiplying by comma values is faster they switched it around. For ease of porting I think it's best we don't rename the variable.